app/views/phases/_submissions_table_rows.html.erb

```diff
@@ -1,154 +1,133 @@
+          <span>Yes</span>
+        </div>
       <% else  %>
-          <div class="mobile-lg:display-none">
-              <%= image_tag(
-                  "images/usa-icons/star_outline.svg",
-                  class: "usa-icon--size-3 margin-right-1",
-                  alt: ""
-              )%>
-              <span class="text-top">Not Selected to Advance</span>
-          </div>    
+        <div class="mobile-lg:display-none margin-top-1">
+          <span>No</span>
+        </div>    
       <% end %>      
     </td>
 
     <td data-label="Assigned Evaluators">
       <% active_assignments = submission.evaluator_submission_assignments.select { |a| ['assigned', 'recused'].include?(a.status) } %>
       <% if active_assignments.any? %>
-        <div class="display-flex flex-column">
+        <div class="display-flex flex-column margin-bottom-2">
           <% active_assignments.each do |assignment| %>
             <div class="margin-bottom-05 display-flex flex-row flex-justify">
               <div class="flex-fill">
-                <span class="<%= assignment.recused? ? 'usa-tag bg-error' : '' %>">
-                  <%= "#{assignment.evaluator.first_name} #{assignment.evaluator.last_name}" %>
-                </span>
+                <%= "#{assignment.evaluator.first_name} #{assignment.evaluator.last_name}" %> - <%= evaluator_score(assignment).formatted_score %>
               </div>
               <div class="text-right">
-                <span class="margin-x-1"><%= evaluator_score(assignment).formatted_score %></span>         
-                <span class="usa-tag padding-x-1 <%= evaluation_status_color(assignment.evaluation_status) %>">
+                <span class="usa-tag padding-x-1 text-center width-15 <%= evaluation_status_color(assignment.evaluation_status) %>" style="min-width: 120px; display: inline-block;">
                   <%= assignment.evaluation_status.to_s.titleize %>
                 </span>
               </div>
             </div>
           <% end %>
         </div>
       <% else %>
-        <span class="font-body-xs text-no-wrap">No evaluators assigned to this submission</span>
+        <div class="margin-bottom-2"
+          <span class="font-body-xs text-no-wrap">None yet</span>
+        </div>
       <% end %>
     </td>
 
     <% average_score = average_score(submission).formatted_score %>
     <td data-label="Average Score" data-score="<%= average_score %>">
-      <%= average_score %><%= "%" if weighted_scoring?(@phase) && average_score.presence != "0" %>
-    </td>
-
-    <td>
-      <div class="display-flex flex-no-wrap grid-row grid-gap-1">
-        <%= link_to submission_path(submission) do %>
-          <button class="usa-button font-body-2xs text-no-wrap margin-y-1">
-            View Submission
-          </button>
-        <% end %>    
-      </div>  
+      <div class="margin-bottom-2">
+        <%= average_score %><%= "%" if weighted_scoring?(@phase) && average_score.presence != "0" %>
```

Comment:
I see it was already like this but I'm still trying to wrap my head around the last part of the conditional `average_score.presence != "0"`. when scoring is not completed yet the average_score is "-", which means this line shows it as "-%"? maybe the conditional should be != "-"?
```suggestion
        <%= average_score %><%= "%" if weighted_scoring?(@phase) && average_score.presence != "-" %>
```

Reply:
@emmabjj thoughts on the above?

Reply:
tested the suggested change above locally and it looks good to me 👍